packages/blockchain-wallet-v4-frontend/src/data/components/simpleBuy/sagas.ts

```diff
@@ -1258,20 +1265,44 @@ export default ({
     }
   }
 
+  // Util function to help match payment method ID
+  // to more details about the bank
+  const getBankInformation = function * (order: SBOrderType) {
+    yield put(actions.components.brokerage.fetchBankTransferAccounts())
+    yield take(FETCH_BANK_TRANSFER_ACCOUNTS_SUCCESS)
+    const bankAccountsR = selectors.components.brokerage.getBankTransferAccounts(
+      yield select()
+    )
+    const bankAccounts = bankAccountsR.getOrElse([])
+    const [bankAccount] = filter(
+      (b: BankTransferAccountType) =>
+        // @ts-ignore
+        b.id === prop('paymentMethodId', order),
+      defaultTo([])(bankAccounts)
+    )
+
+    return bankAccount
+  }
+
   const showModal = function * ({ payload }: ReturnType<typeof A.showModal>) {
     const { cryptoCurrency, orderType, origin } = payload
+    const latestPendingOrder = S.getSBLatestPendingOrder(yield select())
+    const bankAccount = yield call(
+      getBankInformation,
+      latestPendingOrder as SBOrderType
+    )
```

Comment:
we should not call this on this place because if it's fresh new wallet this will fails with error 401 and modal will never opened, fix is here https://github.com/blockchain/blockchain-wallet-v4-frontend/pull/3088/files